### ISSUE TITLE
Rock64 1.5GHz CPU, not 1.296GHz

### ISSUE THREAD

**Issue by D4rkSl4ve** [MichaIng/DietPi#2025]
The Rock64 by Pine64.org has a Quad-core Cortex-A53 up to 1.5GHz CPU; as per their Wiki (http://wiki.pine64.org/index.php/ROCK64_Main_Page)

CPU Frequency as per NetData from the DietPi install:
https://i.imgur.com/vniYEaS.jpg

#### Required Information:
- DietPi version	| v6.13 
- Distro version	| stretch or 9.5
- Kernel version	| Linux Rock64TorBox 4.4.77-rockchip-ayufan-136 #1 SMP Thu Oct 12 09:14:48 UTC 201                  7 aarch64 GNU/Linux
- SBC device		| Rock64 (aarch64)
- Power supply	| ROCK64 / PINEBOOK / PINE H64 5V 3A SWITCHING POWER SUPPLY
- SDcard used		| Samsung 64GB 100MB/s (U3) MicroSDXC EVO Select Memory Card

**Comment by Fourdee** [comment]
@D4rkSl4ve 

Thanks for the report 👍 

The CPU frequency stepping has most likely been tweaked by the kernel creator (4.4.77-rockchip-ayufan-136). Usually, this is for stability reasons.

Regardless, i'll setup a system and verify this is the case.

**Comment by Fourdee** [comment]
Marking as closed in favor of:
https://github.com/Fourdee/DietPi/issues/2026#issuecomment-414389675

**Comment by Fourdee** [comment]
@D4rkSl4ve 

Hi,

I've just updated our image to Ayufan 0.7.9 pre-image. Confirmed 1.3GHz limit:
https://dietpi.com/downloads/images/DietPi_Rock64-ARMv8-Stretch.7z

In this case, it would appear Ayufan has specifically limited the max CPU clock to 1.3GHz. Most likely for improved stability, and/or reduce heat.

As we do not compile our own kernels, and we rely on the pre-images available to us, there is nothing we can do our end to change the limit.
